handlers/pulp_rpm/handlers/rpmtools.py

```diff
@@ -140,7 +159,11 @@ def install(self, names):
                 yb.processTransaction()
             else:
                 yb.progress.set_status(True)
-            return Package.installed(yb.tsInfo)
+            details = Package.installed(yb.tsInfo)
+            affected = Package.affected(details)
+            map(log.info, [INSTALLED % dict(p=p) for p in affected])
+            map(yb.logfile.info, [INSTALLED % dict(p=p) for p in affected])
+            return details
```

Comment:
I considered this but Packages.affected() is used to translate/build data, it seemed inappropriate for it to log this.  There is no guarantee when or how many times this method can be called and if it were called multiple times it would have very undesirable side effects in the log.